src/components/Form/CheckAclPermsDocContainerForm.jsx

```diff
@@ -93,30 +99,37 @@ const CheckAclPermsDocContainerForm = () => {
     }
   };
 
-  const formRowStyle = {
-    margin: '20px 0'
-  };
-
   return (
     <FormSection
       title="Check Permission to Documents Container"
       state={state}
       statusType="Permission status"
       defaultMessage="To be set..."
     >
-      <form onSubmit={handleAclPermission} autoComplete="off">
-        <div style={formRowStyle}>
-          <label htmlFor="set-acl-to">
-            Check permissions to username&apos;s Documents container:{' '}
-          </label>
-          <br />
-          <br />
-          <input id="set-acl-to" size="25" name="setAclTo" {...user} placeholder={selectedUser} />
-        </div>
-        <button disabled={state.processing} type="submit">
-          Check Permission
-        </button>
-      </form>
+      <Box display="flex" justifyContent="center" sx={{ minWidth: 120 }}>
+        <form onSubmit={handleAclPermission} autoComplete="off">
+          <FormControl fullWidth>
+            <TextField
+              id="set-acl-to"
+              name="setAclTo"
+              {...user}
+              placeholder={selectedUser}
+              label="Search username"
+              required
+            />
+            <br />
+            <Button
+              variant="contained"
+              fullWidth
```

Comment:
Just as test I deleted the `fullWidth` here and it stayed the same size. 

*Note: the other forms you have converted I don't believe have `fullWidth` for `<Button>`*

---

But yeah, I deleted all these and the form and it's internals stayed the same size. So, not sure these are necessary - and I still don't really know what is controlling the size.

Reply:
I've made your suggestions except for removing `fullWidth` on `<FormControl>` as those do affect the layout. It's harder to tell with this component but is clearer when trying it on the others. So that one I'd like to keep.